### ISSUE TITLE
Mobafire links don't work anymore!

### ISSUE THREAD

**Issue by tombragas** [Cadrach/lol_ise#66]
Mobafire changed it's site so links arent compatible with the Item Set Editor.
Here's the message I get:
![003af137bef77396792093515272dd48](https://user-images.githubusercontent.com/45015832/48439626-a54d4400-e77e-11e8-9dcb-f4ed6b69365b.png)
Btw I'm using Chrome.

**Comment by QuilavaLP** [comment]
Yeah this has also became a problem for me.
Even though the links are correct it still gave that error.

**Comment by JeritHD** [comment]
Yep I think the most reliable guide-to-League tool is now coming to an end. Or not, if the developer is going to fix everything
